### PR TITLE
security: Document current security practises

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
Security is important to our customers. With this document I'd like to
provide openess to our customers before they roll-out or chose our
product.

### CODE REVIEW

**Comment by ZJvandeWeg** [comment]
Not sure how/where to link this page yet. But first let's get a review on the copy.

File: src/product/security.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Security on the application level will have the same properties for self-managed
+installs as well as FlowForge Cloud.
+
+### Authentication and authorization
+
+Authentication and session management is centrally managed in the FlowForge
+application. There's currently no support for SAML or SSO. There's no support
+for centralized management for creation of teams, users per team, nor their RBAC
+roles.
```
Comment (hardillb) [inline src/product/security.md:14]:
```suggestion
roles.

Epic (https://github.com/flowforge/flowforge/issues/226)
```

File: src/product/security.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+[app.flowforge.com](https://app.flowforge.com), while the Node-RED instances are
+served from \*.flowforge.cloud.
+
+### Data Protection
+
+Data protection has two components: while in-transit and at rest.
+
+#### Data in-transit
+
+Data in transit from external sources is protected with SSL/TLS. Encryption is
```
Comment (hardillb) [inline src/product/security.md:38]:
```suggestion
Data in transit to & from external sources is protected with SSL/TLS. Encryption is
```

File: src/product/security.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+terminated at the edge, both for HTTP and MQTT traffic.
+
```
Comment (hardillb) [inline src/product/security.md:40]:
```suggestion

Story to add internal SSL/TLS (https://github.com/flowforge/flowforge/issues/910)
```

File: src/product/security.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+#### Data at rest
+
+FlowForge Cloud stores data in either our relational database or an object
+store. For both the database and the object storage there's currently no
+encryption at rest configured.
+
```
Comment (hardillb) [inline src/product/security.md:46]: Can't add an issue/epic here just yet as it's in the private CloudProject (https://github.com/flowforge/CloudProject/issues/79)

**Review by hardillb — CHANGES_REQUESTED** [review]
I also don't see anything linking to this yet (which may be intentional)

File: src/product/security.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+#### Data in-transit
+
+Data in transit from external sources is protected with SSL/TLS. Encryption is
+terminated at the edge, both for HTTP and MQTT traffic.
+
+#### Data at rest
+
+FlowForge Cloud stores data in either our relational database or an object
+store. For both the database and the object storage there's currently no
+encryption at rest configured.
```
Comment (ZJvandeWeg) [inline src/product/security.md:45]:
```suggestion
encryption at rest configured.

We're migrating to using encryption at rest for the [database (internal link)](https://github.com/flowforge/CloudProject/issues/79)
and for our [object storage (internal link)](https://github.com/flowforge/CloudProject/issues/80).
```

File: src/product/security.md (hunk 1 above)
Comment (ZJvandeWeg) [reply src/product/security.md:14]:
```suggestion
roles.

This is being investigate, see also the [relevant epic](https://github.com/flowforge/flowforge/issues/226).
```

File: src/product/security.md [hunk 6]
```diff
@@ -13,6 +13,8 @@ application. There's currently no support for SAML or SSO. There's no support
 for centralized management for creation of teams, users per team, nor their RBAC
 roles.
 
+This is being investigate, see also the [relevant epic](https://github.com/flowforge/flowforge/issues/226).
```
Comment (ZJvandeWeg) [inline src/product/security.md:16]:
```suggestion
This is being investigated, see also the [relevant epic](https://github.com/flowforge/flowforge/issues/226).
```

File: src/product/security.md (hunk 3 above)
Comment (ZJvandeWeg) [reply src/product/security.md:42]:
```suggestion

Adding internal SSL/TLS is [being investigated](https://github.com/flowforge/flowforge/issues/910).
```

**Pull request closed without merging**